.github/workflows/merge.yml

```diff
@@ -183,6 +195,14 @@ jobs:
         env:
           cmd_type: UNIT_TEST
 
+      - name: Blackduck Scan
+        uses: lumada-common-services/gh-composite-actions@1.2.0
+        env:
+          BlackDuck_Project_Version: ${{ steps.version.outputs.current-version }}
+          BlackDuck_Project_Name: "${{ inputs.blackduck_project_key }}"
+          BlackDuck_Api_Token: "${{ secrets.BLACKDUCK_TOKEN }}"
```

Comment:
There is a `BLACKDUCK_API_TOKEN` in [Pentaho's org secrets](https://github.com/organizations/pentaho/settings/secrets/actions)